tests/api/data/test_submission_remove_submission_other_missing.json

```diff
@@ -0,0 +1,10 @@
+{
+    "endpoint": "submission/remove/submission",
```

Comment:
Incorrect endpoint.

Reply:
What's wrong with the endpoint? It matches the one I defined in api/submission/routes in the server. Do you prefer submission/remove instead?

Reply:
Did you run the tests on you side?
I believe this is causing failing CI.

I would make sure that the routes match the api/user family.

Reply:
The tests that are failing on my end seem to be test_timestamp_conversions, test_nested, and test_simple.

Reply:
1) I would check and make sure you are using a valid version of Python.

2) You should always make sure all tests are passing. When starting work on a new project, that is one of the very first things you can do (you can't be sure you are not breaking things if tests are not run).
You should pause development on api/submission/remove until you have tests passing.